### ISSUE TITLE
Exception when enabling app via CLI

### ISSUE THREAD

**Issue by aszlig** [SergeyMosin/Appointments#29]
Exception and stack trace, which happens during `occ app:enable`:

```
TypeError: Argument 1 passed to OC\Accounts\AccountManager::getAccount() must be an instance of OCP\IUser, null given, called in .../apps/appointments/lib/InstallHook.php on line 36 and defined in .../lib/private/Accounts/AccountManager.php:338
Stack trace:
#0 .../apps/appointments/lib/InstallHook.php(36): OC\Accounts\AccountManager->getAccount(NULL)
#1 [internal function]: OCA\Appointments\InstallHook->__construct('appointments', NULL, Object(OC\AllConfig), Object(OC\Accounts\AccountManager), Object(OC\User\Manager))
#2 .../lib/private/AppFramework/Utility/SimpleContainer.php(82): ReflectionClass->newInstanceArgs(Array)
#3 .../lib/private/AppFramework/Utility/SimpleContainer.php(99): OC\AppFramework\Utility\SimpleContainer->buildClass(Object(ReflectionClass))
#4 .../lib/private/AppFramework/Utility/SimpleContainer.php(114): OC\AppFramework\Utility\SimpleContainer->resolve('OCA\\Appointment...')
#5 .../lib/private/AppFramework/DependencyInjection/DIContainer.php(414): OC\AppFramework\Utility\SimpleContainer->query('OCA\\Appointment...')
#6 .../lib/private/ServerContainer.php(129): OC\AppFramework\DependencyInjection\DIContainer->queryNoFallback('OCA\\Appointment...')
#7 .../lib/private/Repair.php(114): OC\ServerContainer->query('OCA\\Appointment...')
#8 .../lib/private/legacy/app.php(970): OC\Repair->addStep('OCA\\Appointment...')
#9 .../lib/private/Installer.php(158): OC_App::executeRepairSteps('appointments', Array)
#10 .../core/Command/App/Enable.php(110): OC\Installer->installApp('appointments')
#11 .../core/Command/App/Enable.php(85): OC\Core\Command\App\Enable->enableApp('appointments', Array, Object(Symfony\Component\Console\Output\ConsoleOutput))
```

The [call of `IUserManager::get`](https://github.com/SergeyMosin/Appointments/blob/079f55f947ba37781002eb22dd0ef9848d311ad8/lib/InstallHook.php#L32) can potentially return `null` but [`IAccountManager::getAccount`](https://github.com/nextcloud/server/blob/5aaa8a8b5815a24245da6c7a91efb2564492c13f/lib/public/Accounts/IAccountManager.php#L67) doesn't allow the parameter to be `null`able.

I'm not sure why it's even necessary to get the current user account for this app nor do I actually use the app, but this is what showed up in my CI test (full log [here](https://headcounter.org/hydra/log/59c0jyqd1wh3566r89aipf355lla7m6k-vm-test-run-nextcloud-upgrade.drv)) which tries to check updates of as many Nextcloud apps as possible.

**Comment by SergeyMosin** [comment]
fixed in v1.1.6
Account info is need to for improved UX so the users do not have to fill out their email and phone# twice for the public page and for email notificatioins. They can also change it later in the app settings.

Thanks for reporting.

**Comment by aszlig** [comment]
Thanks for fixing (fixed by 67c31aa90c90fa16372fd15b4cac754bd0c3735d).
